infrastructure/prime-app-template.yml

```diff
@@ -999,9 +977,24 @@ objects:
           - name: ${SVC_NAME}-redis-data
             emptyDir: {}
         containers:
-          - resources:
+          - name: redis
+            image: >-
+              image-registry.openshift-image-registry.svc:5000/${OC_LICENSE_PLATE}-tools/redis-6:latest
```

Comment:
Maybe consistency-  but I also see a lot of ${{secrets.OPENSHIFT_IMAGE_REPOSITORY}} in these PRs. Should we have them all the same?

Reply:
`secrets.OPENSHIFT_IMAGE_REPOSITORY` is a GitHub actions-only thing. Having it in the OpenShift will make them anywhere else.